### ISSUE TITLE
Error with click comming episodes

### ISSUE THREAD

**Issue by rmilyard** [SiCKRAGETV/sickrage-issues#2711]
Error !

KeyError: 2

% if cur_result['imdb_id']: <a href="${anon_url('http://www.imdb.com/title/', cur_result['imdb_id'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="http://www.imdb.com/title/${cur_result['imdb_id']}"><img alt="[imdb]" height="16" width="16" src="${sbRoot}/images/imdb.png" /> % endif <a href="${anon_url(sickbeard.indexerApi(cur_indexer).config['show_url'], cur_result['showid'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="${sickbeard.indexerApi(cur_indexer).config['show_url']}"><img alt="${sickbeard.indexerApi(cur_indexer).name}" height="16" width="16" src="${sbRoot}/images/${sickbeard.indexerApi(cur_indexer).config['icon']}" /></a> <span><a href="${sbRoot}/home/searchEpisode?show=${cur_result['showid']}&amp;season=${cur_result['season']}&amp;episode=${cur_result['episode']}" title="Manual Search" id="forceUpdate-${cur_result['showid']}" class="epSearch forceUpdate"><img alt="[search]" height="16" width="16" src="${sbRoot}/images/search16.png" id="forceUpdateImage-${cur_result['showid']}" /></a></span> </span> </div> <span class="title">Next Episode:</span> <span>${'S%02iE%02i' % (int(cur_result['season']), int(cur_result['episode']))} - ${cur_result['name']}</span> 

/usr/pbi/sickrage-amd64/share/sickrage/SickRage/sickbeard/indexers/indexer_api.py, line 38:

return indexerConfig[self.indexerID]

/usr/pbi/sickrage-amd64/share/sickrage/SickRage/gui/slick/views/comingEpisodes.mako, line 419:

<a href="${anon_url(sickbeard.indexerApi(cur_indexer).config['show_url'], cur_result['showid'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="${sickbeard.indexerApi(cur_indexer).config['show_url']}"><img alt="${sickbeard.indexerApi(cur_indexer).name}" height="16" width="16" src="${sbRoot}/images/${sickbeard.indexerApi(cur_indexer).config['icon']}" /></a>

/usr/pbi/sickrage-amd64/share/sickrage/SickRage/gui/slick/views/layouts/main.mako, line 217:

/usr/pbi/sickrage-amd64/share/sickrage/SickRage/lib/mako/runtime.py, line 853:

callable_(context, *args, **kwargs)

**Comment by rmilyard** [comment]
Here is log:

    2015-09-08 19:28:48 Thread-14 :: Failed doing webui callback: Traceback (most recent call last):
  File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/sickbeard/webserve.py", line 272, in async_call
    result = function(**kwargs)
  File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/sickbeard/webserve.py", line 715, in index
    t = PageTemplate(rh=self, file="home.mako")
  File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/sickbeard/webserve.py", line 113, in __init__
    super(PageTemplate, self).__init__(*args, **kwargs)
  File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/lib/mako/template.py", line 321, in __init__
    module = self._compile_from_file(path, filename)
  File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/lib/mako/template.py", line 379, in _compile_from_file
    module = compat.load_module(self.module_id, path)
  File "/usr/pbi/sickrage-amd64/share/sickrage/SickRage/lib/mako/compat.py", line 66, in load_module
    return imp.load_source(module_id, path, fp)
ImportError: Loaded module _usr_pbi_sickrage_amd64_share_sickrage_SickRage_gui_slick_views_home_mako not found in sys.modules

**Comment by OmgImAlexis** [comment]
Looks like a duplicate of https://github.com/SiCKRAGETV/sickrage-issues/issues/2710

**Comment by flightlevel** [comment]
And possibly duplicate of https://github.com/SiCKRAGETV/sickrage-issues/issues/2700

@rmilyard Can you stop sickrage.> Go to Config - Help and Info to find your cache folder
In the cache folder delete all the folders except for the images folder.
Start SR and report back

**Comment by rmilyard** [comment]
Still same error after removing all folders

**Comment by didyouexpectthat** [comment]
I just got this as well. 
https://sickrage/comingEpisodes/

Mako Runtime error
```
Error !

KeyError: 2

% if cur_result['imdb_id']: <a href="${anon_url('http://www.imdb.com/title/', cur_result['imdb_id'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="http://www.imdb.com/title/${cur_result['imdb_id']}"><img alt="[imdb]" height="16" width="16" src="${sbRoot}/images/imdb.png" /> % endif <a href="${anon_url(sickbeard.indexerApi(cur_indexer).config['show_url'], cur_result['showid'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="${sickbeard.indexerApi(cur_indexer).config['show_url']}${cur_result['showid']}"><img alt="${sickbeard.indexerApi(cur_indexer).name}" height="16" width="16" src="${sbRoot}/images/${sickbeard.indexerApi(cur_indexer).config['icon']}" /></a> </td> <td align="center"> <a href="${sbRoot}/home/searchEpisode?show=${cur_result['showid']}&amp;season=${cur_result['season']}&amp;episode=${cur_result['episode']}" title="Manual Search" id="forceUpdate-${cur_result['showid']}x${cur_result['season']}x${cur_result['episode']}" class="forceUpdate epSearch"><img alt="[search]" height="16" width="16" src="${sbRoot}/images/search16.png" id="forceUpdateImage-${cur_result['showid']}" /></a> </td>
/usr/share/sickrage/sickbeard/indexers/indexer_api.py, line 38:
return indexerConfig[self.indexerID]
/usr/share/sickrage/gui/slick/views/comingEpisodes.mako, line 266:
<a href="${anon_url(sickbeard.indexerApi(cur_indexer).config['show_url'], cur_result['showid'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="${sickbeard.indexerApi(cur_indexer).config['show_url']}${cur_result['showid']}"><img alt="${sickbeard.indexerApi(cur_indexer).name}" height="16" width="16" src="${sbRoot}/images/${sickbeard.indexerApi(cur_indexer).config['icon']}" /></a>
/usr/share/sickrage/gui/slick/views/layouts/main.mako, line 217:
/usr/share/sickrage/lib/mako/runtime.py, line 853:
callable_(context, *args, **kwargs)
```

**Comment by flightlevel** [comment]
@didyouexpectthat 
Did you try without https or clearing your cache?

**Comment by didyouexpectthat** [comment]
I did clear the cache and it did not resolve the issue. Turning off SSL did not resolve this specific issue, either.

**Comment by nivfreak** [comment]
Same problem here: 

Error !

KeyError: 2

% if cur_result['imdb_id']: <a href="${anon_url('http://www.imdb.com/title/', cur_result['imdb_id'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="http://www.imdb.com/title/${cur_result['imdb_id']}"><img alt="[imdb]" height="16" width="16" src="${sbRoot}/images/imdb.png" /> % endif <a href="${anon_url(sickbeard.indexerApi(cur_indexer).config['show_url'], cur_result['showid'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="${sickbeard.indexerApi(cur_indexer).config['show_url']}"><img alt="${sickbeard.indexerApi(cur_indexer).name}" height="16" width="16" src="${sbRoot}/images/${sickbeard.indexerApi(cur_indexer).config['icon']}" /></a> <span><a href="${sbRoot}/home/searchEpisode?show=${cur_result['showid']}&amp;season=${cur_result['season']}&amp;episode=${cur_result['episode']}" title="Manual Search" id="forceUpdate-${cur_result['showid']}" class="epSearch forceUpdate"><img alt="[search]" height="16" width="16" src="${sbRoot}/images/search16.png" id="forceUpdateImage-${cur_result['showid']}" /></a></span> </span> </div> <span class="title">Next Episode:</span> <span>${'S%02iE%02i' % (int(cur_result['season']), int(cur_result['episode']))} - ${cur_result['name']}</span>

/data/home/mark/devel/echel0n/SickRage/sickbeard/indexers/indexer_api.py, line 38:
return indexerConfig[self.indexerID]

/data/home/mark/devel/echel0n/SickRage/gui/slick/views/comingEpisodes.mako, line 419:
<a href="${anon_url(sickbeard.indexerApi(cur_indexer).config['show_url'], cur_result['showid'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="${sickbeard.indexerApi(cur_indexer).config['show_url']}"><img alt="${sickbeard.indexerApi(cur_indexer).name}" height="16" width="16" src="${sbRoot}/images/${sickbeard.indexerApi(cur_indexer).config['icon']}" /></a>

/data/home/mark/devel/echel0n/SickRage/gui/slick/views/layouts/main.mako, line 217:

/data/home/mark/devel/echel0n/SickRage/lib/mako/runtime.py, line 853:
callable_(context, *args, **kwargs)

**Comment by papampi** [comment]
after last update I'm getting same error on upcoming episode page: 

```
Error !

KeyError: 2

% if cur_result['imdb_id']: <a href="${anon_url('http://www.imdb.com/title/', cur_result['imdb_id'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="http://www.imdb.com/title/${cur_result['imdb_id']}"><img alt="[imdb]" height="16" width="16" src="${sbRoot}/images/imdb.png" /> % endif <a href="${anon_url(sickbeard.indexerApi(cur_indexer).config['show_url'], cur_result['showid'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="${sickbeard.indexerApi(cur_indexer).config['show_url']}${cur_result['showid']}"><img alt="${sickbeard.indexerApi(cur_indexer).name}" height="16" width="16" src="${sbRoot}/images/${sickbeard.indexerApi(cur_indexer).config['icon']}" /></a> </td> <td align="center"> <a href="${sbRoot}/home/searchEpisode?show=${cur_result['showid']}&amp;season=${cur_result['season']}&amp;episode=${cur_result['episode']}" title="Manual Search" id="forceUpdate-${cur_result['showid']}x${cur_result['season']}x${cur_result['episode']}" class="forceUpdate epSearch"><img alt="[search]" height="16" width="16" src="${sbRoot}/images/search16.png" id="forceUpdateImage-${cur_result['showid']}" /></a> </td>
/opt/sickrage/sickbeard/indexers/indexer_api.py, line 38:
return indexerConfig[self.indexerID]
/opt/sickrage/gui/slick/views/comingEpisodes.mako, line 266:
<a href="${anon_url(sickbeard.indexerApi(cur_indexer).config['show_url'], cur_result['showid'])}" rel="noreferrer" onclick="window.open(this.href, '_blank'); return false" title="${sickbeard.indexerApi(cur_indexer).config['show_url']}${cur_result['showid']}"><img alt="${sickbeard.indexerApi(cur_indexer).name}" height="16" width="16" src="${sbRoot}/images/${sickbeard.indexerApi(cur_indexer).config['icon']}" /></a>
/opt/sickrage/gui/slick/views/layouts/main.mako, line 217:
/opt/sickrage/lib/mako/runtime.py, line 853:
callable_(context, *args, **kwargs)
```

**Comment by rmilyard** [comment]
I have cleared cache.  Rebooted etc.  Still same issues.  Also looks like shows for yesterday didn't download.

**Comment by papampi** [comment]
fixed on mine with latest commit 9edd012e9ab013265b3631706e3aea5673146ae4

**Comment by faidwen** [comment]
Good show, fixed with latest.
